okapi-core/src/main/java/org/folio/okapi/service/impl/ProcessModuleHandle.java

```diff
@@ -70,6 +70,21 @@ public ProcessModuleHandle(Vertx vertx, LaunchDescriptor desc, String id,
     }
   }
 
+  private Future<Void> waitPortOpen(NetClient c, int iter) {
+    return c.connect(port, "localhost")
+        .compose(socket ->
+                socket.close().compose(y -> {
+                  if (iter == 0) {
+                    return Future.failedFuture(
+                        messages.getMessage("11502", Integer.toString(port)));
+                  }
+                  Promise<Void> promise = Promise.promise();
+                  vertx.setTimer(100, x -> waitPortOpen(c, iter - 1)
+                      .onComplete(promise::handle));
```

Comment:
```suggestion
                  vertx.setTimer(100, x -> waitPortOpen(c, iter - 1).onComplete(promise::handle));
```
Avoid the misleading line break.

Reply:
OK. Allow me to put waitPortOpen on its own line instead.